src/queryBuilder.ts

```diff
@@ -70,7 +73,11 @@ export function GenerateQuery<T extends object>(
 	const bindings: unknown[] = [];
 	switch (type) {
 		case QueryType.SELECT: {
-			query = `SELECT * FROM \`${tableName}\``;
+			let columns = "*";
+			if (options.columns && options.columns.length > 0) {
+				columns = options.columns.map((x) => `\`${String(x)}\``).join(", ");
```

Comment:
Can we use `"` (double quotes) for this instead, as is the standard for column names?